libtiledbsoma/src/utils/arrow_adapter.h

```diff
@@ -841,11 +843,250 @@ class ArrowAdapter {
         }
     }
 
-    static void set_current_domain_slot(
-        tiledb_datatype_t type,
-        const void* buff,
-        NDRectangle& ndrect,
-        std::string name);
+    template <typename T, size_t Take, size_t Skip = 0>
+    static std::vector<std::array<T, Take>> get_table_column_by_name(
```

Comment:
This doesn't appear to be used anywhere. Can you pull it out into a separate PR and/or add it to the branch where it is used?